sites/shared/hooks/use-apikey-docs.mjs

```diff
@@ -0,0 +1,30 @@
+// Hooks
+import { useCallback } from 'react'
+import { useDynamicMdx } from 'shared/hooks/use-dynamic-mdx.mjs'
+// Components
+import { MdxWrapper } from 'shared/components/wrappers/mdx.mjs'
+
+export const useApikeyDocs = (locale) => {
+  const loader = (option) =>
+    useCallback(
+      () =>
+        import(
+          /* webpackInclude: /docs\/site\/apikeys\/[a-z]+\/[a-z][a-z]\.md$/ */ `markdown/docs/site/apikeys//${option}/${locale}.md`
+        ),
+      [locale]
+    )
+
+  const docs = {}
+
+  for (const option of ['name', 'expiry', 'level']) {
+    const { MDX, frontmatter } = useDynamicMdx(loader(option))
```

Comment:
`useDynamicMdx` may be a a bad name for that hook. It is meant for use with mdx that we're dynamically loading *on page load*, so it's technically dynamic, but it's not wrapped in `next/dynamic` because it wasn't really necessary for the implementation. 

These docs hooks really want to lazy-load this markdown, so for that I recommend taking a look at [the dynamic org docs implementation](https://github.com/freesewing/freesewing/blob/f0721700fdcf477d42440db2fc5d8d43e10c1d2c/sites/shared/components/dynamic-docs/org.mjs#L25), though maybe we should unify the approach

Reply:
My focus in these changes was on presenting a more consistent experience to the user when inputting data.
So how we load the MDX was not something I was into. I just want to make sure that ever input has contextual (modal) help.

As per your suggestion, I have removed these various hooks and instead just use the dynamic docs component.
However, it seems that the custom components are not working. Which is odd because I'm pretty certain they did work.

Anyway, this might require another look but for now I'm leaving it as is.

Reply:
understood. I can take a look if that would help